index.js

```diff
@@ -30,7 +30,7 @@ const action = async context => {
 };
 
 const gifski = {
-	title: 'Export With Gifski',
+	title: 'Export with Gifski',
```

Comment:
This was intentional. I've been meaning to open a PR to Kap docs about it being title case. That's what macOS uses too.

Reply:
Makes sense. We should probably change the default save file service and open PRs for the other plugins